### DISCUSSION TITLE
Shrink the height(margin) of the status bar?

### DISCUSSION THREAD

**Discussion by mozbugbox** [lwindolf/liferea#991] [General]
The `status bar` feels too bulky compare to the `menu bar`.

With the Gtk widget inspector, it shows that the statusbar itself has a margin of 8 while menubar has 0. At the same time, the GtkBox on left of the status bar itself has a margin of 4. So overall, there is are 12px margin for the status bar. Or 24px if you add up both the top and bottom. 

I think the margins for top and bottom of the statusbar can be reduced to 4 or 2 to save some precious vertical space. At the same time make the status bar look less bulky.

**Comment by sunwire** [comment]
I agree. In addition, height changes minimally when the text is displayed.

**Comment by lwindolf** [comment]
Saving vertical space always sounds right :-)

**Reply by lwindolf** [reply]
Implemented with #1152 to be released with 1.14-RC2
